python/runtime/cudaq/platform/py_alt_launch_kernel.cpp

```diff
@@ -99,10 +99,10 @@ jitAndCreateArgs(const std::string &name, MlirModule module,
     PassManager pm(context);
     pm.addNestedPass<func::FuncOp>(cudaq::opt::createPySynthCallableBlockArgs(
         SmallVector<StringRef>(names.begin(), names.end())));
+    pm.addPass(cudaq::opt::createLambdaLiftingPass());
```

Comment:
The reason this was done after GKE was to keep the size of the embedded MLIR string small. I think that still holds true, since lambda lifting can cause code growth.

We probably need to make sure lambda lifting is run in all the JIT pipelines.

Reply:
I can move it back and make PyRemoteRestQPU.cpp run it as it done before.